layout.py

```diff
@@ -10,10 +15,10 @@ def __init__(self, size, pos=Pos(0,0), border=0):
 
     def _render_children(self, surface):
         for child in self.children:
-            child.render(surface)
+            child.draw(surface)
 
-    def add(self, child):
+    def add_element(self, child):
```

Comment:
For consistency, `Layout` is also inherits the sprite class. This makes it so that drawing and updating them can be done altogether with one command (look at `pyKiwi-game.py`'s game loop).